op-batcher/batcher/tx_data.go

```diff
@@ -9,14 +9,23 @@ import (
 	"github.com/ethereum-optimism/optimism/op-service/eth"
 )
 
+// DaType determines how txData is submitted to L1.
+type DaType int
+
+const (
+	DaTypeCalldata DaType = iota
```

Comment:
consider adding docs for these constants

Reply:
[f3f784a](https://github.com/Layr-Labs/optimism/pull/22/commits/f3f784ab7dadac9ad29cff0d2307bbc40d0d3fc7)